src/main/java/net/minecraftforge/coremod/api/ASMAPI.java

```diff
@@ -80,17 +91,17 @@ public static MethodInsnNode buildMethodCall(final String ownerName, final Strin
     }
 
     /**
-     * Signifies the type of number constant for a {@link LdcNumberType}.
+     * Signifies the type of number constant for a {@link NumberType}.
      */
-    public enum LdcNumberType {
+    public enum NumberType {
```

Comment:
Why was this renamed/does it break compatibility? How old is this?

Reply:
It was renamed because it is no longer exclusive to `LdcInsnNode`. There aren't any versions of Forge that have bumped CoreMods recently to include `LdcNumberType`, so it doesn't break compatibility.

Reply:
Okay, then if you're done i can pull this

Reply:
Yeah it's ready